app.js

```diff
@@ -41,6 +41,12 @@ app.get('/crit/:crit_id', (req, res) => {
 	});
 	
 });
+//welcome page route
+app.get('/Welcome', (req, res) => {
+	res.render('NEWUSER');})
+        //route for profile
+app.get('/Profile', (req, res) => {
+	res.render('profile');})

```

Comment:
Same comment here: lowercase your "P" and add a newline after `res.render('profile');` so that your curly bracket and paren are on the next line. Add a semi-colon.